mkosi.default.d/fedora/10-mkosi.fedora

```diff
@@ -65,6 +65,7 @@ BuildPackages=
         /usr/bin/xsltproc
 
 Packages=
+        acl
```

Comment:
Hmm, CI still complains:

```
/usr/lib/systemd/systemd: error while loading shared libraries: libip4tc.so.2: cannot open shared object file: No such file or directory
Error: Process completed with exit code 127.
```

I guess we should either drop `pkgconfig(libiptc)` from the build `BuildPackages=` or add `iptables-legacy` to the `Packages=`.

Reply:
Thanks. libiptc is now dropped.